tasks/configure-Debian.yml

```diff
@@ -0,0 +1,17 @@
+- stat:
+    path: /etc/init.d/orientdb
+  register: service_orientdb_stat
+
+- name: Install orientdb as a service
+  copy:
+    src="{{orientdb_path}}/bin/orientdb.sh"
```

Comment:
The copy module does not work here, because it considers the "src" value on the host machine. 
The previous workaround (command module to wrap a 'cp') was there to copy from remote host to remote host. Ansible 2.0 brings the "remote_src" variable to the "copy" module but to keep backward compatibility (it is 1.6 in meta informations for galaxy), we cannot use it.